interface/patient_file/report/custom_report.php

```diff
@@ -484,12 +484,7 @@ function next_prev(action){
   echo '<page_header style="text-align:right;" class="custom-tag"> ' . xlt("PATIENT") . ':' . text($titleres['lname']) . ', ' . text($titleres['fname']) . ' - ' . $titleres['DOB_TS'] . '</page_header>    ';
   echo '<page_footer style="text-align:right;" class="custom-tag">' . xlt('Generated on') . ' ' . oeFormatShortDate() . ' - ' . text($facility['name']) . ' ' . text($facility['phone']) . '</page_footer>';
 
-  // Use logo if it exists as 'practice_logo.png' in the site dir
-  // old code used the global custom dir which is no longer a valid
-   $practice_logo = "$OE_SITE_DIR/images/practice_logo.png";
-   if (file_exists($practice_logo)) {
-        echo "<img src='$practice_logo' align='left'><br />\n";
-     } 
+  echo "<img src='../../../sites/default/images/practice_logo.png' align='left' style='width : 45px ; height : 56px'><br>";
```

Comment:
Look at your logo size and original logo size (1800 x 2913px). 
The height is too small.

Reply:
when you add it with the original size the image will occupy the whole page and will make it look bad